x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/email/email_params.test.tsx

```diff
@@ -17,13 +17,24 @@ describe('EmailParamsFields renders', () => {
       subject: 'test',
       message: 'test message',
     };
+
+    const connector = {
+      secrets: {},
+      config: {},
+      id: 'test',
+      actionTypeId: '.test',
+      name: 'Test',
+      isPreconfigured: false,
+    };
+
     const wrapper = mountWithIntl(
       <EmailParamsFields
         actionParams={actionParams}
         errors={{ to: [], cc: [], bcc: [], subject: [], message: [] }}
         editAction={() => {}}
         index={0}
         docLinks={{ ELASTIC_WEBSITE_URL: '', DOC_LINK_VERSION: '' } as DocLinksStart}
+        actionConnector={connector}
```

Comment:
We should have this `actionConnector` as an optional property and remove this changes on the files where it not used to request the connector